packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -211,14 +211,20 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
   }
 
   /** Get ValidatorGroup information */
-  async getValidatorGroup(address: Address): Promise<ValidatorGroup> {
+  async getValidatorGroup(
+    address: Address,
+    getAffiliates: boolean = true
+  ): Promise<ValidatorGroup> {
     const res = await this.contract.methods.getValidatorGroup(address).call()
     const accounts = await this.kit.contracts.getAccounts()
     const name = (await accounts.getName(address)) || ''
-    const validators = await this.getRegisteredValidators()
-    const affiliates = validators
-      .filter((v) => v.affiliation === address)
-      .filter((v) => !res[0].includes(v.address))
+    let affiliates: Validator[] = []
```

Comment:
preexisting, nit: we tend to use `members` elsewhere and in docs, right?

Reply:
Yeah but these are the ones that are affiliated to the group that are specifically not members